doc/api/fs.md

```diff
@@ -1419,6 +1427,8 @@ changes:
   * `encoding` {string|null} **Default:** `'utf8'`
   * `mode` {integer} **Default:** `0o666`
   * `flag` {string} See [support of file system `flags`][]. **Default:** `'a'`.
+  * `mkdirp` {boolean} "mkdir -p" directories in `path` if they do not exist.
```

Comment:
why don't we call this flag `parents` (which is what the `-p` of `mkdir`) stands for.

Reply:
good idea.  i didn't know it was shorthand for --parent until now.